### PR TITLE
[CIN-2001] Update authentication request to Tara

### PR DESCRIPTION
**Pull request by surabhyaaryal**
https://snapdocs-eng.atlassian.net/browse/CIN-2001

### CODE REVIEW

File: postman/Public Collection.postman_environment.json [hunk 1]
```diff
@@ -21,17 +21,23 @@
...
 			"value": "<string>",
 			"type": "text",
 			"enabled": true
 		},
+		{
+			"key": "audience",
+			"value": "<url string>",
+			"type": "default",
+			"enabled": true
+		},
```
Comment (surabhyaaryal) [inline postman/Public Collection.postman_environment.json:40]: Screen Shot 2022-09-22 at 8 17 48 AM

File: postman/Snapdocs Closings API - Public Collection.postman_collection.json [hunk 2]
```diff
@@ -46,7 +47,7 @@
 								"header": [],
 								"body": {
 									"mode": "raw",
-									"raw": "{\n    \"grant_type\": \"client_credentials\",\n    \"client_id\": \"{{ClientID}}\",\n    \"client_secret\": \"{{ClientSecret}}\",\n    \"scope\": \"borrower_activities:index users:index transaction_statuses:index closings:create closings:external_look_up closings:show closings:mark_finished_uploading_docs documents:create documents:index documents:show subscriptions:create subscriptions:destroy subscriptions:index borrowers:index\"\n}",
+									"raw": "{\n    \"grant_type\": \"client_credentials\",\n    \"client_id\": \"{{ClientId}}\",\n    \"client_secret\": \"{{ClientSecret}}\",\n    \"audience\": \"{{audience}}\"\n}",
 									"options": {
```
Comment (surabhyaaryal) [inline postman/Snapdocs Closings API - Public Collection.postman_collection.json:51]: Screen Shot 2022-09-22 at 8 18 03 AM

File: postman/Public Collection.postman_environment.json (hunk 1 above)
Comment (drbobstone) [reply postman/Public Collection.postman_environment.json:40]: nice, thanks!

Addition to this request - can we add the ENV variable values to be csdemo0 values?  prepopulate the baseurl, url, and audience values?  (i guess add audience as an Env Var too)

Comment (surabhyaaryal) [reply postman/Public Collection.postman_environment.json:40]: will do.

**Review by drbobstone — APPROVED** [review]
Looks great!

**Pull request merged**